scripts/Tools/code_checker

```diff
@@ -57,10 +57,16 @@ def run_pylint(on_file):
 ###############################################################################
     pylint = find_executable("pylint")
 
-    cmd = "%s --disable=I,C,R,logging-not-lazy,wildcard-import,unused-wildcard-import,fixme,broad-except,bare-except,eval-used,exec-used,global-statement %s" % (pylint, on_file)
+    cmd_options = " --disable=I,C,R,logging-not-lazy,wildcard-import,unused-wildcard-import,fixme,broad-except,bare-except,eval-used,exec-used,global-statement"
+
+    if "scripts/Tools" in on_file:
```

Comment:
Why remove relative import warning? It's actually useful because it helps reduce the cases where the tools can't find imports because you ran them from a different location.